File: python_programming/chapter5/random_words.rb
words = ["OVERUSED", "CLAM", "GUAM", "TAFFETA", "PYTHON"]
new_words = []

while words.length() > 0
	choice = rand(words.length())
	word_choice = words[choice]
	print(word_choice + "\n")
	if new_words.include?(word_choice)
		next
	else
		new_words << word_choice
		words.delete(word_choice)
	end
end

File: python_programming/chapter5/random_words.py
import random

words = ["OVERUSED", "CLAM", "GUAM", "TAFFETA", "PYTHON"]
new_words = []

while len(words) > 0:
	choice = random.randrange(len(words))
	word_choice = words[choice]
	print(word_choice)
	if word_choice in new_words:
		next
	else:
		new_words.append(word_choice)
		del words[choice]

print(new_words)
